packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_waiting.spec.ts

```diff
@@ -0,0 +1,12 @@
+context('cy.origin waiting', () => {
+  beforeEach(() => {
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="cross-origin-secondary-link"]').click()
+  })
+
+  it('.wait()', () => {
+    cy.origin('http://foobar.com:3500', () => {
+      cy.wait(500)
```

Comment:
Are we wanting to actually test that this waited? Or are we happy enough with it not throwing an error?

Reply:
I [updated the test](https://github.com/cypress-io/cypress/pull/21137/commits/2f1dfde1c649b7b27334f54cbfa4bc8f5b6442a2) to verify things a bit better. It also won't needlessly wait for 500ms.